apps/github-api-integration/README.md

```diff
@@ -0,0 +1,212 @@
+API (Github) Integration
+========================
+
+This example demonstrates how to integrate a web form with a remote API using Home Office Forms (HOF).
+Specifically, this demonstrates how to integrate a HOF form with the Github API. However, this example only uses Github as the remote API in order to provide a potentially real-life example of API integration. With some small tweaks, pertaining to the url, and quite possibly the Model, this example could be repurposed to fit any number of simple API integrations.
+
+If you'd rather get started quickly [skip to install & start](#install-and-start)
+
+# Tutorial
+
+The three main components to this example are the Github model, the behaviours and the apps' `index.js` that configures the specfics of the app.
+
+To begin by writing the Model means when we introduce new components to the stack our upstream dependencies are known quantaties and therefore easier to reason about.
+
+## Models
+
+### Github model
+
+Our Github Model will extend from [hof-model](https://github.com/UKHomeOfficeForms/hof-model).
+```js
+const Model = require('hof-model');
+
+module.exports = class GithubModel extends Model {}
+```
+
+### Setting options at instantiation
+
+We know that [all Github API requests require a User-Agent header](https://developer.github.com/v3/#user-agent-required) so we can start by overriding the parent Model's `constructor` method to ensure all requests carry the correct header.
+
+```js
+constructor(attributes, options) {
```

Comment:
No, that's fair, however, I'm treading a thin line between explaining everything in fine detail and staying on point.